test/cmd/deployments.sh

```diff
@@ -47,11 +47,7 @@ tryuntil oc get rc/test-deployment-config-1
 # tryuntil oc get rc/test-deployment-config-2
 
 # scale rc via deployment configuration
-oc scale dc test-deployment-config --replicas=1
-oc scale dc test-deployment-config --replicas=2 --timeout=10m
-# scale directly
-oc scale rc test-deployment-config-1 --replicas=4
-oc scale rc test-deployment-config-1 --replicas=5 --timeout=10m
+oc scale dc test-deployment-config --replicas=4
```

Comment:
Makes sense... Although I hate to keep bringing in PRs to this one- I'd rather get something we're pretty comfortable with into master and iterate on enhancing the tests.